src/components/Navbar/Navbar.jsx

```diff
@@ -78,9 +75,18 @@ function Navbar() {
           <small>Cart</small>
         </div>
 
-        <NavLink to="/login">
-          <button className="btn nav-auth-btn-style">Login</button>
-        </NavLink>
+        {token ? (
+          <button
+            className="btn nav-auth-btn-style"
+            onClick={() => handleLogout()}
```

Comment:
you can use onclick without arrow function also

Reply:
Yahh I can , its just out of habit